### PR TITLE
*: mute useless log

### PR DESCRIPTION
**Pull request by BusyJay**
We don't care about the pending status changes if not for debugging. Stale epoch logs are useless too as it shows nothing we can't get from client log or metrics, and it can pollute the log file every time split happens.

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

**Comment by siddontang** [comment]
LGTM

File: src/raftstore/store/peer.rs [hunk 1]
```diff
@@ -655,7 +655,7 @@ impl Peer {
                 if progress.matched >= truncated_idx {
                     let (_, pending_after) = self.peers_start_pending_time.swap_remove(i);
                     let elapsed = duration_to_sec(pending_after.elapsed());
-                    info!(
+                    debug!(
                         "{} peer {} has caugth up logs, elapsed: {}",
```
Comment (Hoverbear) [inline src/raftstore/store/peer.rs]: Can you also fix this spelling mistake `caught`?

**Comment by BusyJay** [comment]
PTAL

**Comment by BusyJay** [comment]
OMG, I'm so tired of updating branch again and again.

**Pull request merged**